test/controllers/ModulesSpec.scala

```diff
@@ -52,11 +63,10 @@ object ModulesSpec extends PlaySpecification with Mockito {
     }
   }
 
-  def createModules: (Modules, ModulesLookup, ModuleDao) = {
-    val lookup = mock[ModulesLookup]
-    val dao = mock[ModuleDao]
-    val modules = new Modules(lookup, dao)
+  def createModules(app: play.api.Application): (Modules, ModulesLookup, ModuleDao) = {
+    val lookup = app.injector.instanceOf[ModulesLookup]
```

Comment:
You could still test with mocks if you only used the injector to get `ControllerComponents`. Perhaps in this case we don't really care much, but it would more closely match the original behavior of the test.